core/src/main/nl/tudelft/ti2806/riverrush/domain/entity/River.java

```diff
@@ -10,66 +13,68 @@
 import com.badlogic.gdx.scenes.scene2d.actions.SequenceAction;
 import com.google.inject.Inject;
 
-import static com.badlogic.gdx.scenes.scene2d.actions.Actions.forever;
-import static com.badlogic.gdx.scenes.scene2d.actions.Actions.sequence;
-
 public class River extends Actor {
 
-    private static final int END_REGIONX = 570;
-    private static final int END_REGIONY = 570;
+  private static final int END_REGIONX = 1599; // 570;
```

Comment:
Not really, but considering its a visual change that hadn't been reviewed I figured it might be best to keep the old values in case we prefer the old ones